src/lib/errorStatusCode.ts

```diff
@@ -0,0 +1,14 @@
+// please do not change the status code numbers
+// you can add new ones but not changing the existing ones
+
+export const errorStatusCode = {
+  1000: "command fails to execute",
```

Comment:
nit: the numbers can also be mapped to an enum or a constat, VALIDATION_ERROR, EXECUTION_ERROR, ENVIRONMENT_ERROR, GIT_ERROR, COMMAND_FAIL_ERROR

Reply:
accept. will make the change. thanks